index.ts

```diff
@@ -8,9 +8,11 @@ svgTags.delete('iframe');
 svgTags.delete('script');
 svgTags.delete('video');
 
-type Attributes = JSX.IntrinsicElements['div'];
-type DocumentFragmentConstructor = typeof DocumentFragment;
-type ElementFunction = ((props?: any) => HTMLElement | SVGElement) & {
+type Attributes = JSX.IntrinsicElements['div'] & {
+	children?: Node[];
+};
+
+type FunctionComponent = ((props?: any) => Element | DocumentFragment) & {
```

Comment:
Since we already install `@types/react`, maybe we can just use `React.FC` instead of this type.
Maybe the `declare global` block below fixes the return type of React.FC to be a DOM element as well.

Reply:
Nope, doesn't work